### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by DanUshindi**
Here are the steps followed to make this project:

  -Create a new GitHub repo.

  -Create a new branch according to the rules.

  -Add a .gitignore file.

  -Set up  html and css linters

  - index.html file that includes a header with the text "Hello Microverse!". 

  -style.css file that includes one class with styles for the header

- test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (elmar8287) [inline .gitignore]: - [ ] Please, consider to add `test.md` file into `.gitignore` file. You can see, that actually it is in your repository and not ignored, as required.
Screen Shot 2022-02-08 at 14 30 32

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+ Getting Started:
+ -Install linters
+ -setup index.html and style.css
+ -write the html code 
+ -write the css code
+
+ bust_in_silhouette Author1:
+ -GitHub:https://github.com/DanUshindi
+ -Twitter:@dan_ushindi
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider using the Microverse template for the README file. The README file is one of the necessary files, which tell visitors of your repo what is about the project and how to work with it - it makes your project more professional. Here is [link](https://github.com/microverseinc/readme-template) for template

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @DanUshindi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+test.md
```
Comment (elmar8287) [inline .gitignore:2]: - [ ] Perfect! You added `test.md`, bit you can see this file still in your repository. I think you did not delete it first, before pushing changes. Please, first delete `test.md` file from your repository and then push your changes.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @DanUshindi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @DanUshindi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**